x-pack/plugins/security_solution/server/lib/detection_engine/types.ts

```diff
@@ -9,4 +9,4 @@ import type { Filter } from '@kbn/es-query';
 
 export type PartialFilter = Partial<Filter>;
 
-export type RefreshTypes = false | 'wait_for';
+export type RefreshTypes = boolean | 'wait_for';
```

Comment:
Nit: since we're no longer using `wait_for` this could be simplified: 
```suggestion
export type RefreshTypes = boolean;
```

Probably best to narrow the type as much as possible (unless we want to allow `wait_for` in the near future, which judging by https://github.com/elastic/kibana/issues/163953 we do not).